atomisticparsers/gromacs/parser.py

```diff
@@ -774,12 +774,51 @@ def get_mdp_file(self):
 
         return self.get_gromacs_file(self.mdp_ext)
 
-    def get_gromacs_file(self, ext):
+    def get_gromacs_file(self, ext, return_all=False):
```

Comment:
I don't love that this function is returning completely different data structures depending on what it finds. I know that it was like that before, but there are not so many instances calling this function, I think we can standardize the return and then properly type it. Maybe we just discuss further when we meet...